sound/soc/sof/intel/cnl.c

```diff
@@ -303,6 +303,7 @@ const struct snd_sof_dsp_ops sof_cnl_ops = {
 	/* dsp core power up/down */
 	.core_power_up = hda_dsp_enable_core,
 	.core_power_down = hda_dsp_core_reset_power_down,
+	.core_get = hda_dsp_core_get,
```

Comment:
what about APL? is there any reason to prevent this addition?
We know the firmware does not support multicore but since the flows are identical why not set this for APL as well, so that IF we ever change the firmware the kernel is ready.